src/Calculations.cpp

```diff
@@ -115,3 +84,64 @@ void Calculations::calculateLJF() {
     }
   }
 }
+
+
+void Calculations::LCcalculateLJF(std::vector<Particle*> &center, std::vector<Particle> &other) {
+  if (center.size() > 1) {
+    calculateLJFcenter(center);
+  }
+  if (other.size() > 0) {
+    for (size_t i = 0; i < center.size(); ++i) {
```

Comment:
```suggestion
    for (auto pi : center) {
```

If you don't need the index inside the loop, range based for loops are preferred, since they read nicer and are less error prone.